src/app/view/createLogo.js

```diff
@@ -0,0 +1,19 @@
+import logo from '/static/assets/img/logo.png'
```

Comment:
if logo.png is not showing in our app change route to:
`../../../../static/assets/img/logo.png`